### ISSUE TITLE
Endre til ffe-farge-vann-30 for headinger og lead, dark mode

### ISSUE THREAD

**Issue by andrea-sb1** [SpareBank1/designsystem#1715]
### Beskrivelse
Headinger og lead har ffe-farge-vann-70 i dark mode. Disse ivaretar gjeldende lovkrav for kontrast, WCAG 2.0 krav 1.4.3 (nivå AA), når de brukes på bakgrunnsfargene for dark mode.

For å sørge for at kontrasten også er god nok til å ivareta WCAG-kontrastnivå AAA, samt ny og mer presis måte å beregne fargekontrast på som vil bli introdusert i versjon 3.0 av WCAG (APCA), kan vi endre fargen på headingene og lead til ffe-farge-vann-30.

Forslaget ble presentert i designforum 22.11.2023, for tilbakemeldinger fra resten av designerne. Det ble blant annet vist eksempler på hvordan løsningene i nettbanken/mobilbanken vil se ut med headinger og lead i farge vann-30.

[Mer om lovkravet for kontrast, WCAG 1.4.3, nivå AA (ekstern lenke)](https://www.uutilsynet.no/wcag-standarden/143-kontrast-minimum-niva-aa/95)
[Mer om WCAG 1.4.6, nivå AAA (ekstern lenke)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced.html)
[Mer om APCA (ekstern lenke)](https://www.accessibility.com/glossary/advanced-perceptual-contrast-algorithm)

<img width="696" alt="Eksempel med headinger og lead i vann 70 og vann 30" src="https://github.com/SpareBank1/designsystem/assets/77671758/4d26f345-2270-4ceb-93f3-0833d1dd5d3c">

### Kommentar
Det pågår en ny diskusjon rundt dette på Slack, så vi bør avvente litt med å implementere denne endringen :)

**Comment by raglie-sb1** [comment]
Er det ikke bedre å gjøre endringen for å komme innenfor lovkravet? Også endre det på nytt når de blir enige? 
(hvor kan jeg eventuelt følge diskusjonen @andrea-sb1?)

**Comment by raglie-sb1** [comment]
Arkiverer denne da den vil bli fikset med fargevariabler, og opp til visuell faggruppe hvilke farger som blir brukt hvor.
